### ISSUE TITLE
Number of Model Parameters?

### ISSUE THREAD

**Issue by spark157** [gitter-lab/nn4dms#1]
Hello,

I was wondering if you had handy the number of trainable model parameters in each of the models?  Thanks.

Scott

**Comment by samgelman** [comment]
Hey Scott, 

```
avgfp lr  number of trainable parameters: 9,481
avgfp fc  number of trainable parameters: 968,401
avgfp cnn number of trainable parameters: 3,118,409
avgfp gcn number of trainable parameters: 3,077,065
bgl3  lr  number of trainable parameters: 20,041
bgl3  fc  number of trainable parameters: 2,014,301
bgl3  cnn number of trainable parameters: 1,573,993
bgl3  gcn number of trainable parameters: 1,605,993
gb1   lr  number of trainable parameters: 2,241
gb1   fc  number of trainable parameters: 2,242,001
gb1   cnn number of trainable parameters: 747,081
gb1   gcn number of trainable parameters: 858,953
pab1  lr  number of trainable parameters: 3,001
pab1  fc  number of trainable parameters: 320,401
pab1  cnn number of trainable parameters: 990,281
pab1  gcn number of trainable parameters: 242,793
ube4b lr  number of trainable parameters: 4,081
ube4b fc  number of trainable parameters: 428,401
ube4b cnn number of trainable parameters: 1,390,409
ube4b gcn number of trainable parameters: 1,381,961
```

I also added a Jupyter notebook showing how to access the trainable variables/parameters as an example.

-Sam

**Comment by agitter** [comment]
Thanks for the interest in our work @spark157.  Please let us know if you have any questions about the new notebook.

Closed by f1b78809c4f63fe103246ff84cc953cf7e763edd
